pkgs/by-name/je/jellyseerr/package.nix

```diff
@@ -54,6 +54,11 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postBuild
   '';
 
+  postBuild = ''
+    # Clean up broken symlinks left behind by `pnpm prune`
+    pnpm store prune
```

Comment:
I've tested this locally (x86_64-linux), but it does not seem to fix the issue. Weird that this started happening now, it did not happen when I first updated the package.

Reply:
Maybe it's best to take the approach of e.g. https://github.com/NixOS/nixpkgs/pull/378937 and disable the warning, if it all seems to work?

Reply:
Another option could be removing broken symlinks in node modules (and if it's a monorepo, packages/**/node_modules) like in https://github.com/NixOS/nixpkgs/pull/380354/files